root/static/scripts/relationship-editor/utility/mergeDates.js

```diff
@@ -26,13 +26,13 @@ export default function mergeDates(
   ) {
     return null;
   }
-  /* eslint-disable @stylistic/js/no-multi-spaces */
+
   const mergedDate: PartialDateT = {
     day:   ad ?? bd ?? null,
     month: am ?? bm ?? null,
     year:  ay ?? by ?? null,
   };
-  /* eslint-enable @stylistic/js/no-multi-spaces */
```

Comment:
I'm curious why this is relevant to the eslint v9 package upgrade specifically if the stylistic rules are in a separate package, and nothing from `@stylistic` was updated in yarn.lock here -- or was this meant to be in a different commit?

Reply:
That's because of this bit of the commit message:

```
In some places, eslint let us know the rules we were disabling are no longer
erroring (probably due to now fixed eslint bugs). As such, we can remove
those disable lines.

```

Reply:
Yeah, I read that part of the commit but I think I misunderstood what it was implying:

I had thought the `@stylistic/js/no-multi-spaces` rule was part of the `@stylistic/eslint-plugin` package rather than `eslint` package, so was wondering why upgrading the latter would cause this (apparently) unrelated rule to no longer error.

What I seem to have misunderstood is that the feature to warn about unused "disable" directives is itself a new feature, and these specific directives were *already* unused before the commit.

Reply:
Yeah, that's what I was trying to say, maybe not very clearly :sweat_smile:

Reply:
You could say something like, "as a new feature, ESLint now lets us know where rules we are disabling are no longer warning or erroring."

Reply:
I reworded the commit to say something to that effect :)